app/templates/components/wds-global-navigation/anon.hbs

```diff
@@ -0,0 +1,14 @@
+{{#wds-dropdown class='wds-global-navigation__user-menu' hasShadow=true as |dropdown|}}
+	{{#dropdown.toggle as |chevron|}}
+		{{wds-avatar}}{{chevron}}
+	{{/dropdown.toggle}}
+	{{#dropdown.content class='wds-global-navigation__dropdown-content wds-is-right-aligned'}}
```

Comment:
dropdownRightAligned=true

Reply:
I think `.wds-global-navigation__dropdown-content` is never used in scss